api/routes/user.js

```diff
@@ -5,12 +5,13 @@ var SpotifyWebApi = require('spotify-web-api-node');
 var scopes = ['user-read-private', 'user-read-email', 'user-read-birthdate', 'user-top-read', 'user-library-read',
 'playlist-modify-private', 'playlist-read-private', 'playlist-modify-public','user-read-recently-played'],
   state = 'spotification-state';
-var configSpotify = require('./config-spotify');
+var configSpotify = require('../config-spotify');
 var spotifyApi = new SpotifyWebApi(configSpotify);
 
 
 const jwt = require('jsonwebtoken');
-var jwtSecret = require('./config-jwt');
+// var jwtSecret = require('../config-jwt');
+var jwtSecret = 'keepitasecret';
```

Comment:
Totally forgot to give you guys the config files, I'll add to to the group chat. The reason I don't think we should be doing this is that it's probably good practice to not have api/secret keys on the GitHub.

Reply:
Yeah for sure, we can use either gitignores or environment variables